tools/data_manipulation/read_netcdf/read_netcdf/netcdf_metadata_info.xml

```diff
@@ -0,0 +1,77 @@
+<tool id="netcdf_metadata_info" name="Netcdf Metadata Info" version="0.1.0">
+    <description>summarize content of a nc file</description>
+    <requirements>
+        <requirement type="package" version="1.1.6">netcdf-metadata-info</requirement> 
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        netcdf-metadata-info '$input'
+        &&
+        while read -r l;do
+            a=\$(echo \$l | cut -d' ' -f1);echo \$l>dimensions_\$a
+        ;done <variables.tabular
+        &&
+        rm dimensions_VariableName 
+        &&
+        for f in dimensions_*; do cat \$f | sed 's/ /\t\n/g' | sed '\$s/$/ /' >\$f.tabular ; done
+        &&
+        for f in dimensions_*.tabular;do
+            awk 'NR % 2 != 0' \$f > \$f.2
+            &&
+            sed 1d \$f.2 > \$f 
+            &&
+            rm \$f.2
+        ;done
+        &&
+        ncdump -h '$input' > '$info'
+    ]]></command>
+    <inputs>
+        <param type="data" name="input" label="Netcdf file" format="netcdf,h5" help="Netcdf file you need information about."/>
+    </inputs>
+    <outputs>
+<!--
+        <data name="var_tabs" format="tabular">
```

Comment:
why is this commented out ... if the conda_activation log was the problem ... simply create a new folder `mkdir results`, put all your results into this folder and discover your outputs from this new folder

Reply:
Don't remember why, just removed it!